components/InputField.js

```diff
@@ -34,16 +34,17 @@ function FieldGroup({ label, help, pre, post, after, button, className, ...props
   props.key = props.key || props.name;
 
   const inputProps = { ...props };
+  const id = uniqueId('styled-input-group');
```

Comment:
This is problematic because:
1. The component will have a different `id` after each render (so whenever the value changes)
2. Server-side rendering and client-side rendering will eventually produce different results, which can cause more problems later (https://github.com/facebook/react/issues/11336#issuecomment-338617882)

Since `InputField` is marked as deprecated, I would argue for not doing anything for accessibility here and ignore the errors for this file.

Reply:
@Betree : Thanks for the info, I agree. I had the particular rule disabled for this file. 😄